zoo-prometheus/prometheus.yml

```
@@ -0,0 +1,38 @@
+global:
+  scrape_interval: 60s
+
+scrape_configs:
+  - job_name: prometheus
+    static_configs:
+    - targets:
```

yamllint

[warning] wrong indentation: expected 6 but found 4

(indentation)